cla_public/apps/contact/fields.py

```diff
@@ -76,7 +80,9 @@ def process_formdata(self, valuelist):
 
     @classmethod
     def _format(cls, value):
-        return '{:%Y%m%d}'.format(value)
+        if value and not isinstance(value, basestring):
```

Comment:
wanted to keep FormattedChoiceField generic enough so that if someone wanted to use it for something else that needs formatting they can just implement a get_format_string method. (not assumed to be a format string for a datetime) and if it's already a string then don't do anything.